### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by jacobherrington**
French is my second language, so please make corrections if necessary!

### CODE REVIEW

File: locales/fr.json [hunk 1]
```diff
@@ -114,8 +117,8 @@
 	"Default View": "Vue par défaut",
 	"Compressed View": "Vue compressée",
 	"Search": "Chercher",
-	"Blog Type": "Blog Type",
-	"Blog Address": "Blog Address",
+	"Blog Type": "Type de blog",
+	"Blog Address": "Adresse de blog",
```
Comment (daiyam) [inline locales/fr.json]: better: "Blog Address": "Adresse du blog",

File: locales/fr.json [hunk 2]
```diff
@@ -114,8 +117,8 @@
 	"Default View": "Vue par défaut",
 	"Compressed View": "Vue compressée",
 	"Search": "Chercher",
-	"Blog Type": "Blog Type",
-	"Blog Address": "Blog Address",
+	"Blog Type": "Type de blog",
```
Comment (daiyam) [inline locales/fr.json]: better: "Blog Type": "Type du blog",

File: locales/fr.json (hunk 1 above)
Comment (jacobherrington) [reply locales/fr.json]: Thanks!

File: locales/fr.json (hunk 2 above)
Comment (jacobherrington) [reply locales/fr.json]: Thanks!

File: locales/fr.json [hunk 3]
```diff
@@ -154,5 +158,9 @@
 	"Convert textual arrows to beautiful signs. ⚠ This will interfere with using HTML comments in your Markdown.": "Convertir des flèches textuelles en jolis signes. ⚠ Cela va interferérer avec les éventuels commentaires HTML dans votre Markdown.",
 	"⚠ You have pasted a link referring an attachment that could not be found in the storage location of this note. Pasting links referring attachments is only supported if the source and destination location is the same storage. Please Drag&Drop the attachment instead! ⚠": "⚠ Vous avez collé un lien qui référence une pièce-jointe qui n'a pas pu être récupéré dans le dossier de stockage de la note. Coller des liens qui font référence à des pièces-jointes ne fonctionne que si la source et la destination et la même. Veuillez plutôt utiliser du Drag & Drop ! ⚠",
 	"Enable smart table editor": "Activer l'intelligent éditeur de tableaux",
-	"Snippet Default Language": "Langage par défaut d'un snippet"
+	"Snippet Default Language": "Langage par défaut d'un snippet",
+	"New Snippet": "Nouveau snippet",
+	"Custom CSS": "CSS personnalisé",
+	"Snippet name": "Nom de le snippet",
```
Comment (daiyam) [inline locales/fr.json]: correct: "Snippet name": "Nom du snippet",

File: locales/fr.json [hunk 4]
```diff
@@ -154,5 +158,9 @@
...
+	"Snippet prefix": "Préfixe de le snippet"
```
Comment (daiyam) [inline locales/fr.json]: correct: "Snippet prefix": "Préfixe du snippet"

**Comment by Rokt33r** [comment]
@jacobherrington Could you fix the conflicts? I'm going to merge this pr before this weekend. So I can include this to the next release.

**Comment by jacobherrington** [comment]
Done @Rokt33r!

**Comment by Rokt33r** [comment]
Awesome!

**Pull request merged**